inductiva/fluids/scenarios/coastal_area/bathymetry.py

```diff
@@ -250,9 +250,9 @@ def plot(
         The data is interpolated from the points where the bathymetry is defined
         to the uniform grid using linear interpolation.
 
-        Points on the uniform grid at a distance to points where the bathymetry
-        is defined larger than a threshold distance `max_distance` are not
-        plotted.
+        Points on the uniform grid at a distance larger than a threshold
+        distance `max_distance` from points where the bathymetry is defined are
```

Comment:
Then, shouldn't it be `distance_threshold`?

Reply:
It could be, yes! Thanks. I'll update this in a new PR.